lib/curses.cpp

```diff
@@ -31,6 +31,11 @@ static std::vector<chtype> chtypes_from_numbers(const std::vector<Number> &chstr
 
 namespace rtl {
 
+Number curses$KEY_F(Number c)
```

Comment:
This doesn't need to be a native function. In the future when the compiler can identify pure functions, it won't be able to do that for native functions. So this should be implemented in neon.